### PR TITLE
Task 3 7 implement ui for entering a game

### PR DESCRIPTION
**Pull request by mariusalexander**

### CODE REVIEW

File: page/enter_game.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  page: page
+}
+
+function page (req, res) {
+  console.log('making enter game page...')
+
+  var fs = require('fs')
+  fs.readFile('html/enter_game.html', 'utf8', function (err, html) {
+    if (err) throw err
+    res.end(html)
```
Comment (DavidRisch) [inline page/enter_game.js]: Please use  `html_creator`

File: js/enter_game.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+module.exports = {
```
Comment (DavidRisch) [inline js/enter_game.js]: Files in `js/` are not node modules, they should not contain `module.exports`

File: api/api.js [hunk 3]
```diff
@@ -50,6 +50,11 @@ function init (app) {
         break
       }
 
+      case 'enterGame': {
+        require('../js/enter_game').process(body.playerA1, body.playerA2, body.playerB1, body.playerB2, body.goalsA, body.goalsB)
```
Comment (DavidRisch) [inline api/api.js]: Api methods should be located in the `api/`directory

**Review by DavidRisch — CHANGES_REQUESTED** [review]
Please also look at the changes I made in 456b994d and cf2215e6

**Comment by mariusalexander** [comment]
ready to merge?

**Pull request merged**